tests/generation/test_generation_tf_logits_process.py

```diff
@@ -101,7 +106,8 @@ def test_temperature_dist_warper(self):
         self.assertGreater(tf.math.reduce_max(probs[1, :]), tf.math.reduce_max(warped_prob_smooth[1, :]))
         self.assertLess(tf.math.reduce_min(probs[1, :]), tf.math.reduce_min(warped_prob_smooth[1, :]))
 
-    def _get_repetition_penalty_inputs(self):
+    @parameterized.expand([(False,), (True,)])
+    def test_repetition_penalty_dist_process(self, use_xla):
```

Comment:
changed this one from a two-test format (added in the previous PR) to the parametrized format